src/include/common/container/concurrent_vector.h

```diff
@@ -125,6 +125,11 @@ class ConcurrentVector {
    */
   Iterator end() { return Iterator(vector_.end()); }  // NOLINT for STL name compability
 
+  /**
+   * @return size of the vector.
+   */
+  uint32_t size() const { return vector_.size(); }  // NOLINT for STL name compability
+
```

Comment:
Not sure why this is added in this PR when you don't appear to use `concurrent_vector` at all.

Reply:
oops that is a rellic of days long past